vm/classes/classes.go

```diff
@@ -1,22 +1,23 @@
 package classes
 
 const (
-	ObjectClass    = "Object"
-	ClassClass     = "Class"
-	IntegerClass   = "Integer"
-	FloatClass     = "Float"
-	StringClass    = "String"
-	ArrayClass     = "Array"
-	HashClass      = "Hash"
-	BooleanClass   = "Boolean"
-	NullClass      = "Null"
-	ChannelClass   = "Channel"
-	RangeClass     = "Range"
-	MethodClass    = "method"
-	PluginClass    = "Plugin"
-	GoObjectClass  = "GoObject"
-	FileClass      = "File"
-	RegexpClass    = "Regexp"
-	MatchDataClass = "MatchData"
-	GoMapClass     = "GoMap"
+	ObjectClass         = "Object"
+	ClassClass          = "Class"
+	IntegerClass        = "Integer"
+	FloatClass          = "Float"
+	StringClass         = "String"
+	ArrayClass          = "Array"
+	HashClass           = "Hash"
+	ConcurrentHashClass = "ConcurrentHash"
```

Comment:
`ConcurrentHash` is kind of a library, so it shouldn't have a constant here. Here's only for builtin classes (sorry I should clarify this in comment).

Reply:
And I prefer `Concurrent::Hash` over `ConcurrentHash`, are you ok with this?

Reply:
Makes sense, I will change it :+1: